configs/full.yaml

```diff
@@ -99,10 +92,27 @@ npz_fixed_field_keys:
 # key_mapping:
 #   free_energy: total_energy
 
+# npz example 
+# the keys used need to be stated at least once in key_mapping, npz_fixed_field_keys or include_keys
+# key_mapping is used to map the key in the npz file to the NequIP default values (see data/_key.py)
+# all arrays are expected to have the shape of (nframe, natom, ?) except the fixed fields
+# note that if your data set uses pbc, you need to also pass an array that maps to the nequip "pbc" key
+dataset: npz                                                                       # type of data set, can be npz or ase
+dataset_url: http://quantum-machine.org/gdml/data/npz/toluene_ccsd_t.zip           # url to download the npz. optional
+dataset_file_name: ./benchmark_data/toluene_ccsd_t-train.npz                       # path to data set file
+key_mapping:
+  z: atomic_numbers                                                                # atomic species, integers
+  E: total_energy                                                                  # total potential eneriges to train to
+  F: forces                                                                        # atomic forces to train to
+  R: pos                                                                           # raw atomic positions
+npz_fixed_field_keys:                                                              # fields that are repeated across different examples
+  - atomic_numbers
+
 # A list of atomic types to be found in the data. The NequIP types will be named with the chemical symbols, and inputs with the correct atomic numbers will be mapped to the corresponding types.
```

Comment:
@Linux-cpp-lisp maybe a bit unclear what it means that "inputs with the correct atomic numbers will be mapped"

Reply:
```suggestion
# A list of chemical species found in the data. The NequIP atom types will be named after the chemical symbols and ordered by atomic number in ascending order.
# (Type 0 will be named H and and type 1 C in this case.)
# Atoms in the input will be assigned NequIP atom types according to their atomic numbers.
```
clearer?